modules/net/src/endpoint.cpp

```diff
@@ -124,20 +99,21 @@ auto getPortBt(const std::vector<std::byte>& address) -> std::uint16_t {
 }  // namespace
 
 auto Endpoint::port() const -> std::uint16_t {
-  switch (family()) {
-    case AF_INET:
+  switch (type()) {
+    case heph::net::EndpointType::IPV4:
       return getPortIpV4(address_);
-    case AF_INET6:
+    case heph::net::EndpointType::IPV6:
       return getPortIpV6(address_);
-    case AF_BLUETOOTH:
+    case heph::net::EndpointType::BT:
       return getPortBt(address_);
     default:
```

Comment:
nit. it is better not to have default so if you add a new case the compiler will complain if you forget to address it somewhere

Reply:
good point.